.github/workflows/build.yml

```diff
@@ -22,7 +22,5 @@ jobs:
       #   uses: ScribeMD/docker-cache@0.3.4
       #   with:
       #     key: docker-${{ runner.os }}-${{ hashFiles('evm/Dockerfile') }}
-      - name: Forge Build evm/contracts
-        run: make forge-build
```

Comment:
we don't actually need to do this, we have tests that cover if our contracts are out of date.